bin/install_windows_10_sdk.ps1

```diff
@@ -1,5 +1,44 @@
 # Install the Windows 10 SDK and Visual Studio Build Tools.
 
+<#
+.SYNOPSIS
+    Installs Windows 10 SDK and Visual Studio Build Tools for a specific Windows 10 SDK version.
+
+.DESCRIPTION
+    This script automates the installation of Windows 10 SDK and Visual Studio Build Tools.
+    It reads the desired SDK version from a .windows-10-sdk-version file and validates it against
+    a list of supported versions. The script then downloads and installs the Visual Studio Build
+    Tools with the specified Windows 10 SDK version.
+
+.PARAMETER DryRun
+    If specified, the script will only validate inputs and show what would be installed
+    without actually performing the installation.
+
+.EXAMPLE
+    .\install_windows_10_sdk.ps1
+    Installs the Windows 10 SDK version specified in .windows-10-sdk-version file
+
+.EXAMPLE
+    .\install_windows_10_sdk.ps1 -DryRun
+    Performs a dry run showing what would be installed without actual installation
```

Comment:
Arguably those examples are not worth including in the doc. But to be fair, the `.mdc` rule does [include a mention of "Documentation must include […] examples"](https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/pull/160/files#diff-d923a3ecb1f38262e13511362265f273e1b7936fb72e7ba42292195c68416563R45), so maybe we want to tweak that part of the `.mdc` rule to remove the "example" part of that bullet point (and only keep the "For complex logic, provide detailed examples of different use cases" bullet point to only tell it to generate examples if the logic is complex… which is also a bullet point in the rules we could refine the wording of, to better define for it what we mean by "complex"…